### PR TITLE
Re-add note about unsaved ide changes

### PR DESCRIPTION
**Pull request by Topherhindman**
## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/docs/cloud/migration.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ If your account is scheduled for migration, you will see a banner indicating you
     * New IPs: `52.3.77.232`, `3.214.191.130`, `34.233.79.135`
 2. **User invitations** &mdash; Any pending user invitations will be invalidated during the migration. You can resend the invitations after the migration is complete.
 3. **SSO integrations** &mdash; If you've completed the Auth0 migration, your account SSO configurations will be automatically transferred. If you haven't completed the Auth0 migration, dbt Labs recommends doing that before starting the mult-cell migration to avoid service disruptions.
+4. **IDE sessions** &mdash; Any unsaved changes in the IDE might be lost during the migration process. dbt Labs _strongly_ encourages you to commit all changes in the IDE before your scheduled migration time.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/migration.md]:
```suggestion
4. **IDE sessions** &mdash; Any unsaved changes in the IDE might be lost during migration. dbt Labs _strongly_ recommends committing all changes in the IDE before your scheduled migration time.
```

**Review by mirnawong1 — APPROVED** [review]
hey @Topherhindman thanks so much for this proactive PR ! looks great and just one small suggestion that we're happy to commit for you if you're ok with it! @matthewshaver cc-ing you in here as you've been involved with this doc.

File: website/docs/docs/cloud/migration.md (hunk 1 above)
Comment (ryancharris) [reply website/docs/docs/cloud/migration.md]: @mirnawong1 I like these changes. Thanks for the extra set of 👀 !

**Review by ryancharris — APPROVED** [review]
Approved -- I like Mirna's suggestions.

**Comment by mirnawong1** [comment]
thanks @ryancharris and @Topherhindman ! shipping this useful pr for ya'll! ✨

**Pull request merged**